src/app/shared/card/card.component.html

```diff
@@ -1 +1,9 @@
+<a routerLink="/detail/{{hero.id}}" >
+  <div class="character-image">
+    <img src="{{hero.thumbnail.path}}.{{hero.thumbnail.extension}}" alt="{{hero.name}}">
```

Comment:
can you construct the url from the home page it self? and here it can be "imageURL"